Деньгин Роман

RomanDengin opened the pull request:
Дополнительное задание не делал
[Посмотреть решение](https://rawgit.com/RomanDengin/markup-task-3/master/index.html)

Commit:
text materials and basic markup done

Commit:
hyphens added

Commit:
<br> deleted

Commit:
materials and markup done

Commit:
styles are ready

Commit:
images included

Commit:
fonts included

Commit:
style errors fixed

Commit:
some new styles added

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

savichev-igor commented on index.css:
```diff
@@ -0,0 +1,99 @@
+@font-face
+{
+    font-family: old;
+    src: url('fonts/Carmen.ttf');
```
Я бы ещё проверял, есть ли у пользователя уже этот шрифт с помощью `local`

savichev-igor replied:
И не ограничивался одним лишь форматом

savichev-igor commented on index.css:
```diff
@@ -0,0 +1,99 @@
+
+.topnews
+{
+    float: left;
```
К сожалению, `float` в этой задаче использовать нельзя, так как про него ещё не рассказывали :<

savichev-igor commented on index.html:
```diff
@@ -2,8 +2,173 @@
 <body>
+    <h1>Газета для Марти</h1>
+    <hr>
+    <div class="topnews">
```
Практически любой `div` можно заменить на более семантический тэг - уж тут точно

savichev-igor commented on index.css:
```diff
@@ -0,0 +1,99 @@
+
+.topnews > article
+{
+    column-count: 3;
```
Добавь вендорных префиксов, не у всех последние версии браузеров

savichev-igor reviewed (changes requested):
Газета хоть и выглядит приятно, но соблюдены не все особенности вёрстки того времени

А ещё как мне кажется колонка как-то странно уезжает правая
![image](https://user-images.githubusercontent.com/7279995/32035711-6ceb30e0-ba34-11e7-9721-4af2603b2c67.png)

🍅

Commit:
Сделаны исправления
1) div заменены на семантические тэги
2) вместо float идёт разбиение по колонкам
3) добавлены тэги <q> для цитат
4) добавлены вендорные префиксы для поддержки старых браузеров
5) добавлена проверка наличия у пользователя необходимых шрифтов

Commit:
style fixed

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

RomanDengin commented:
🍏 
поправил замечания

savichev-igor commented on index.css:
```diff
@@ -0,0 +1,114 @@
+@font-face
+{
+    font-family: old;
+    src: local('Carmen'), url('fonts/Carmen.ttf');
```
Я бы ещё формат указывал

savichev-igor commented on index.css:
```diff
@@ -0,0 +1,114 @@
+    border-right: thin solid black;
+    border-left: thin solid black;
+    border-bottom: thin solid black;
+    padding: 10px;
```
`padding: 10px 10px 10px 20px;`

Хоть и громоздко, но в одну строку

savichev-igor commented on index.css:
```diff
@@ -0,0 +1,114 @@
+{
+    font-family: simpleText;
+    text-align: justify;
+    column-rule: 1px black solid;
```
Сначала вендорные правила, потом браузерные

Commit:
Fixed some problems + star

Commit:
style fixed

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

savichev-igor commented:
В следующий раз учитывай, что если кликают по `label`, то `input` тоже должен меняться

⭐️
